content/applications/hr/appraisals/skills_evolution.rst

```diff
@@ -54,48 +57,46 @@ This report can be modified to find specific information by adjusting the :ref:`
 <search/filters>` and :ref:`groupings <search/group>` set in the search bar at the top.
 
 .. image:: skills_evolution/skills-report.png
-   :align: center
    :alt: A report showing all the skills grouped by employee.
 
 .. _appraisals/identify-skills:
 
 Use case: Identify employees with specific skills
 =================================================
 
-Since the :guilabel:`Appraisal Skills Report` organizes all skills by employee, it can be difficult
-to find employees with a specific skill at a specific level. To find these employees, a custom
-filter must be used.
+Since the :guilabel:`Appraisal Skills Report` organizes all skills by month, then employee, it can
+be difficult to find employees with a specific skill at a specific level. To find these employees, a
+custom filter must be used.
 
 In this example, the report is modified to show employees with an expert level of Javascript
 knowledge. To view only those employees, first remove all active filters in the search bar.
 
-Next, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the search bar, then click
-:guilabel:`Add Custom Filter` beneath the :icon:`fa-filters` :guilabel:`Filters` column to load an
-:guilabel:`Add Custom Filter` pop-up window.
+Next, click the :icon:`fa-caret-down` :guilabel:`(Toggle Search Panel)` icon in the search bar, then
+click :guilabel:`Add Custom Filter` beneath the :icon:`fa-filters` :guilabel:`Filters` column to
+load an :guilabel:`Add Custom Filter` pop-up window.
 
 Using the drop-down menu in the first field, select :guilabel:`Skill`. Then, keep the second field
-as-is, and select :guilabel:`Javascript` from the third drop-down menu in the third field.
+as-is (:guilabel:`is in)`, and select :guilabel:`Javascript` from the third drop-down menu in the
+third field.
 
 Next, click :guilabel:`New Rule`, and another line appears. In this second line, select
-:guilabel:`Current Skill Level` for the first drop-down field, leave the second field as-is, then
-select :guilabel:`Expert` for the third drop-down field.
+:guilabel:`Current Level` for the first drop-down field, leave the second field as-is :guilabel:`(is
+in)`, then select :guilabel:`Expert` for the third drop-down field.
+
+After the second rule is added, navigate to the text at the top of the pop-up window that shows
+:guilabel:`Match any` :icon:`fa-caret-down` :guilabel:`of the following rules:`. Click the drop-down
+menu for :guilabel:`any` and change it to :guilabel:`all`.
```

Comment:
I dont think it is necessary to include the colon for this UI text, plus it looks strange with a period following it
```suggestion
:guilabel:`Match any` :icon:`fa-caret-down` :guilabel:`of the following rules`. Click the drop-down
menu for :guilabel:`any` and change it to :guilabel:`all`.
```